code/parachain/frame/pallet-multihop-xcm-ibc/src/lib.rs

```diff
@@ -25,6 +65,7 @@ pub mod pallet {
 	use ibc_primitives::Timeout as IbcTimeout;
```

Comment:
Could you move this to separate `pallet.rs` file? Having all of this indented makes reading the code harder. This is enough code that it might just as well go to separate file rather than inside of a mod block.

Reply:
lib.rs is considered as root in all other pallets.

usually noise moved to types.rs, traits.rs, math.rs. like router part sure should not be in pallet, but not in support.rs. 

this was experiemnt so to allow merge without deep review before it already was merged. it failed tbh. so will not try that next time:)